tests/test_mars_format.py

```diff
@@ -14,10 +13,10 @@
 from datumaro.plugins.mars_format import MarsImporter
 from datumaro.util.test_utils import compare_datasets
 
+from tests.conftest import ASSETS_DIR
 from tests.requirements import Requirements, mark_requirement
 
-ASSETS_DIR = osp.join(osp.dirname(__file__), "assets")
-DUMMY_MARS_DATASET = osp.join(ASSETS_DIR, "mars_dataset")
+DUMMY_MARS_DATASET = str(ASSETS_DIR / "mars_dataset")
```

Comment:
```suggestion
DUMMY_MARS_DATASET = os.fspath(ASSETS_DIR / "mars_dataset")
```
In other places as well.

Reply:
`str(Path(...))` should be sufficient here, IMO. What prompted the suggestion?

Reply:
If the value is supposed to be used as a path in system calls, the `os.fspath` is the correct function to get a string representation.

Reply:
The only utility for `os.fspath` is to be used to get string representation of an arbitrary `PathLike` object. 

Here we're dealing not with arbitrary `PathLike` object, but with `pathlib.PurePath` subclass. 

A canonical way to get `PurePath` string representation is using `str(path)`.